debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -155,14 +155,23 @@
             .withDescription("The unique identifier of the Debezium connector")
             .withNoValidation();
 
+    public static final Field KAFKA_QUERY_TIMEOUT_MS = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "kafka.query.timeout.ms")
+            .withDisplayName("Kafka admin client query timeout (ms)")
+            .withType(Type.LONG)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTION_ADVANCED, 1)) // TODO Validate the group & position
```

Comment:
Please confirm Group and Position. Not sure what are those.

Reply:
@snigdhasjg PLease remove the `TODO`, the value is correct.

Reply:
@snigdhasjg Could you please increase the value by one so the values are unique?

@rk3rn3r Do you know why the collison was not reported?

Reply:
Done. But may I know where is CONNECTION_ADVANCED position 1? Because I couldn’t find it.

Reply:
@snigdhasjg oh, my bad, I misread `ADVANCED` vs `CONNECTION_ADVANCED`. Please use `ADVANCED` as the other one is usually used form setting-up connection to the database. Could also please squash the commits too?

Reply:
@jpechane the duplicate check uses `PostgresConnectorConfig.ALL_FIELDS` to check for duplicates. The new field wasn't added to that list yet.

Reply:
@rk3rn3r I see at it will never be added - as PostgreSQL does not use database history.

Reply:
Good morning 🙈
I fixed my reply. For sure, it should go to the respective `ConnectorConfig` implementation that actually uses the property.

Reply:
@jpechane should @snigdhasjg add the property/field to the connectors that are using database history in order to execute the duplicate check?

Reply:
@rk3rn3r Yes, that is good point. It should go into all connector configs that use daabase history - so `HistorizedRelationalDatabaseConnectorConfig.CONFIG_DEFINITION` and {{Db2ConnectorConfig.configDef}}.

Reply:
> Please use ADVANCED as the other one is usually used form setting-up connection to the database. Could also please squash the commits too?

Done

---
> It should go into all connector configs that use daabase history - so `HistorizedRelationalDatabaseConnectorConfig.CONFIG_DEFINITION` and {{Db2ConnectorConfig.configDef}}

@jpechane I have added the property in `CONFIG_DEFINITION`. But couldn't find anything related `Db2ConnectorConfig`

Reply:
I just found another repo `debezium-connector-db2`
Do you mean [this file](https://github.com/debezium/debezium-connector-db2/blob/main/src/main/java/io/debezium/connector/db2/Db2ConnectorConfig.java#L265-L267)?

Reply:
> Yes, that is good point. It should go into all connector configs that use daabase history - so `HistorizedRelationalDatabaseConnectorConfig.CONFIG_DEFINITION` and `Db2ConnectorConfig.configDef`.

Agreed on the former; but on the latter, we should rather make this one consistent with the other, i.e. have it also expand the config defined in `HistorizedRelationalDatabaseConnectorConfig`. I've logged https://issues.redhat.com/browse/DBZ-4537, and I think we can handle this separately.

Reply:
@jpechane @gunnarmorling @rk3rn3r whats my next step? I have modified `CONFIG_DEFINITION` but haven’t changed anything on `debezium-connector-db2` repo.

Reply:
@snigdhasjg I think @gunnarmorling meant we handle DB2 separately with the mentioned task DBZ-4537. Maybe you could add a comment mentioning that new property so we don't forget to cover it when working on DB2 properties?

Reply:
#DBZ-4537 this card is closed.

@rk3rn3r Sorry for asking, where should I add the comment about new property?

Reply:
@ani-sha did you take care of this when working on DBZ-4537 or else could you file a follow up task for that?

Reply:
@rk3rn3r, so DBZ-4537 is closed already. But could you remind me, what is the follow-up work you have in mind?

Reply:
@gunnarmorling I was referring to 

>  so HistorizedRelationalDatabaseConnectorConfig.CONFIG_DEFINITION and {{Db2ConnectorConfig.configDef}}.

especially the Db2 connector related work

Reply:
@jpechane, ok, could you log a follow-up issue for this then? I've really lost track of what this is about. Thx!